sdk/formrecognizer/Azure.AI.FormRecognizer/tests/SessionRecords/DocumentModelAdministrationClientLiveTests/StartBuildModelWithNeuralBuildModeAsync.json

```diff
@@ -0,0 +1,17097 @@
+{
```

Comment:
Recording file has around 800~900KB. Pretty large for a recording, and is probably caused by the amount of polls we try to do while the long Neural test runs. We could try increasing the polling interval, but I don't think it's large enough to cause concern (also, we have only two recordings like this).

Reply:
I decided not to even test this in live CI because I'm concerned about timeouts.

Reply:
We only have two tests that use Neural so I hope we won't have any issues. Also, we run tests in parallel, so I think these won't cause a big bottleneck.